xradar/model.py

```diff
@@ -995,7 +995,7 @@ def determine_cfradial2_sweep_variables(obj, optional, dim0):
 def conform_cfradial2_sweep_group(obj, optional, dim0=None):
     if dim0 is None:
         # handling first dimension
-        dim0 = "elevation" if obj.sweep_mode.load() == "rhi" else "azimuth"
+        dim0 = "elevation" if obj.sweep_mode.load().astype(str) == "rhi" else "azimuth"
```

Comment:
What error message is displayed which makes this astype necessary?

Reply:
No errors are generated, but the statement is always false because the string is originally represented as b'rhi', for instance. It will only be considered true if you convert it to a regular string or explicitly assign it as b'rhi'.

Reply:
Thanks, we need to keep an eye on that.